src/components/book.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+const Book = () => (
+  <div>
+    <p>Action</p>
+    <h2>The Hanger Games</h2>
+    <p>Suzanne Collins</p>
+    <ul>
+      <li>Comments</li>
+      <li>Remove</li>
+      <li>Edit</li>
+    </ul>
+    <div>64% Completed</div>
+    <div>
+      <p>CURRENT CHAPTER</p>
+      <p>Chapter 17</p>
+      <button type="button">UPDATE PROGRESS</button>
+    </div>
+
+  </div>
+);
```

Comment:
- [ ] This component is expected to receive props of title and author passed to it and then are rendered your h2 for the title of the book and the p tag in line 7 as the author of the book. Kindly go to the bookList where you used the Book component and pass a single book or title and author as a prop to the looped data so the props will be received here and get rendered. 👍 

```jsx
 const Book = (props) => (
  <div>
    <p>Action</p>
    <h2>{props.title}</h2>
    <p>{props.author}</p>
    <ul>
      <li>Comments</li>
      <li>Remove</li>
      <li>Edit</li>
    </ul>
    <div>64% Completed</div>
    <div>
      <p>CURRENT CHAPTER</p>
      <p>Chapter 17</p>
      <button type="button">UPDATE PROGRESS</button>
    </div>

  </div>
);
```